aten/src/ATen/native/native_functions.yaml

```diff
@@ -2738,6 +2743,12 @@
     CPU: gelu_backward_cpu
     CUDA: gelu_backward_cuda
 
+- func: infinitely_differentiable_gelu_backward(Tensor grad, Tensor self) -> Tensor
+  use_c10_dispatcher: full
+  variants: function
+  python_module: nn
```

Comment:
Is that important? This function shout not be exposed to python anyways right?

Reply:
Yup, this doesn't do anything. I put it in to make it consistent with all of the other registered nn backward functions (those have a python_module: nn that doesn't do anything). E.g., 
https://github.com/pytorch/pytorch/blob/367426494759ddde0896665ed55c6f9af2870cf0/aten/src/ATen/native/native_functions.yaml#L2726
https://github.com/pytorch/pytorch/blob/367426494759ddde0896665ed55c6f9af2870cf0/aten/src/ATen/native/native_functions.yaml#L701